vizro-core/docs/pages/development/contributing.md

```diff
@@ -137,7 +140,8 @@ You can easily create such a fragment by running
 hatch run docs:changelog
 ```
```

Comment:
I would seperate here in 2 sentences and start a new one after `hatch run docs:changelog`